### PR TITLE
Fix Craft extension to support Setapp version

### PR DESCRIPTION
**Pull request by 0x77dev**
## Description

Fixes issue #4241 

## Screencast

Screenshot 2023-01-13 at 01 32 35

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/craftdocs/src/Config.ts [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+import { readdirSync, existsSync } from "fs";
 import path from "path";
 
-const craftDataRoot = "~/Library/Containers/com.lukilabs.lukiapp/Data/Library/Application Support/com.lukilabs.lukiapp";
-const searchPath = `${craftDataRoot}/Search`;
+const bundleIds = ["com.lukilabs.lukiapp", "com.lukilabs.lukiapp-setapp"];
+const [craftDataRoot] = bundleIds
+  .map((id) => {
+    return path.join(homedir(), `/Library/Containers/${id}/Data/Library/Application Support/${id}`);
+  })
```
Comment (kudrykv) [inline extensions/craftdocs/src/Config.ts:9]: You can inline it — 
```ts
.map((id) => path.join(homedir(), `/Library/Containers/${id}/Data/Library/Application Support/${id}`))
```

**Review by kudrykv — APPROVED** [review]
❤️

File: extensions/craftdocs/src/Config.ts (hunk 1 above)
Comment (0x77dev) [reply extensions/craftdocs/src/Config.ts]: Done

File: extensions/craftdocs/src/Config.ts [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+import { readdirSync, existsSync } from "fs";
 import path from "path";
 
-const craftDataRoot = "~/Library/Containers/com.lukilabs.lukiapp/Data/Library/Application Support/com.lukilabs.lukiapp";
-const searchPath = `${craftDataRoot}/Search`;
+const bundleIds = ["com.lukilabs.lukiapp", "com.lukilabs.lukiapp-setapp"];
+const [craftDataRoot] = bundleIds
+  .map((id) => path.join(homedir(), `/Library/Containers/${id}/Data/Library/Application Support/${id}`))
+  .filter(existsSync);
+const searchPath = path.join(craftDataRoot, "Search");
```
Comment (thomaspaulmann) [inline extensions/craftdocs/src/Config.ts:9]: What happens if you have both versions installed? Doesn't this make a wrong `searchPath`? 

We have similar extensions where we usually expose a preference as a dropdown for this reason.

**Review by thomaspaulmann — COMMENTED** [review]
Great first contribution! Is there any way to make this a bit more reliable?

**Comment by pernielsentikaer** [comment]
Are you still around @0x77dev

**Comment by 0x77dev** [comment]
Hey @pernielsentikaer @thomaspaulmann,

I pushed an update with application selector preference and updated views accordingly.

**Review by kudrykv — APPROVED** [review]

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋

Looks good to me, approved 🔥

**Pull request merged**